client/layout/masterbar/logged-out.jsx

```diff
@@ -10,6 +10,7 @@ import WordPressWordmark from 'calypso/components/wordpress-wordmark';
 import { isDomainConnectAuthorizePath } from 'calypso/lib/domains/utils';
 import { login } from 'calypso/lib/paths';
 import { addQueryArgs } from 'calypso/lib/route';
+import PluginNavbarHeader from 'calypso/my-sites/plugins/plugin-navbar-header';
```

Comment:
Always loading components that will not be displayed in most of the cases could be a good opportunity for async loading.